src/app/system/global-configurations/edit-configuration/edit-configuration.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="configurationForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label>Configuration Name</mat-label>
+            <input matInput required formControlName="name">
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Configuration Value</mat-label>
+            <input matInput required formControlName="value" [value]="configuration.value">
```

Comment:
Binding with the value is not required as we are using reactive forms.
```suggestion
            <input matInput required formControlName="value">
```